test/e2e/tests/copy_rename.spec.js

```diff
@@ -27,23 +27,23 @@ test('Copy and Rename with Versioned document', async ({ page }, workerInfo) =>
   await page.locator('div.ProseMirror').fill('First text');
 
   // Wait 3 secs to ensure its saved in da-admin
```

Comment:
Comments don't match the time. Is there a way to wait for an event to complete, rather than setting a time out?

Reply:
Good catch on the comment - fixed.

On the Playwright level I haven't found another solution, although you can wait for things to appear, this takes cycles as well. I think that the problem here is that the PW browsers are running in the same GH action environment that just has very low CPU capacity, and I only found that giving the system some breathing space like this works consistently. 
But - if there is a better solution I would be all for it 😄